### PR TITLE
title case and docket no.

### PR DESCRIPTION
**Pull request by klohman**
- changing Docket number to "Docket No." in tables (per Jessica)
- updating table headers to title case

### CODE REVIEW

**Review by matthopson — COMMENTED** [review]
advancedDocumentSearchHelper.js : Line 23
advancedDocumentSearchHelper.test.js : Line 37

**Review by matthopson — APPROVED** [review]

File: web-client/src/views/AdvancedSearch/DocumentSearchResults.jsx [hunk 1]
```diff
@@ -25,13 +25,13 @@ export const DocumentSearchResults = connect(
...
 
-            <table className="usa-table search-results responsive-table row-border-only">
+            <table className="usa-table search-results docket-record responsive-table row-border-only">
               <thead>
                 <tr>
                   <th aria-hidden="true" className="small-column"></th>
                   <th aria-hidden="true" className="small-column"></th>
-                  <th>Docket number</th>
-                  <th>Case title</th>
+                  <th>Docket No.</th>
```
Comment (kkoskelin) [inline web-client/src/views/AdvancedSearch/DocumentSearchResults.jsx]: Any place that we are changing text to `Docket No.` I would like the enclosing tag to use an aria-label. e.g.
`<th aria-label="docket number">Docket No.</th>` for accessibility and screen readers.

File: web-client/src/views/AdvancedSearch/PractitionerSearchResults.jsx [hunk 2]
```diff
@@ -23,10 +23,10 @@ export const PractitionerSearchResults = connect(
             <table className="usa-table search-results docket-record responsive-table row-border-only">
               <thead>
                 <tr>
-                  <th>Bar no.</th>
+                  <th>Bar No.</th>
```
Comment (kkoskelin) [inline web-client/src/views/AdvancedSearch/PractitionerSearchResults.jsx]: *aria-label

File: web-client/src/views/BlockedCasesReport/BlockedCasesReport.jsx [hunk 3]
```diff
@@ -43,10 +43,10 @@ export const BlockedCasesReport = connect(
                     <table className="usa-table row-border-only subsection work-queue deadlines">
                       <thead>
                         <tr>
-                          <th>Docket number</th>
-                          <th>Date blocked</th>
-                          <th>Case title</th>
-                          <th>Case status</th>
+                          <th>Docket No.</th>
```
Comment (kkoskelin) [inline web-client/src/views/BlockedCasesReport/BlockedCasesReport.jsx]: *aria-label

File: web-client/src/views/CaseDeadlines/CaseDeadlines.jsx [hunk 4]
```diff
@@ -68,9 +68,9 @@ export const CaseDeadlines = connect(
                 <table className="usa-table row-border-only subsection work-queue deadlines">
                   <thead>
                     <tr>
-                      <th>Due date</th>
-                      <th>Docket number</th>
-                      <th>Case title</th>
+                      <th>Due Date</th>
+                      <th>Docket No.</th>
```
Comment (kkoskelin) [inline web-client/src/views/CaseDeadlines/CaseDeadlines.jsx]: *aria-label

File: web-client/src/views/CaseInventoryReport/CaseInventoryReport.jsx [hunk 5]
```diff
@@ -107,13 +107,13 @@ export const CaseInventoryReport = connect(
                   <table className="usa-table row-border-only subsection case-inventory">
                     <thead>
                       <tr>
-                        <th>Docket number</th>
-                        <th>Case title</th>
+                        <th>Docket No.</th>
```
Comment (kkoskelin) [inline web-client/src/views/CaseInventoryReport/CaseInventoryReport.jsx]: *aria-label

File: web-client/src/views/Messages/CaseMessagesIndividualCompleted.jsx [hunk 6]
```diff
@@ -13,7 +13,7 @@ export const CaseMessagesIndividualCompleted = connect(
           <thead>
             <tr>
               <th className="small" colSpan="2">
-                Docket Number
+                Docket No.
```
Comment (kkoskelin) [inline web-client/src/views/Messages/CaseMessagesIndividualCompleted.jsx:16]: *aria-label

File: web-client/src/views/Messages/CaseMessagesIndividualInbox.jsx [hunk 7]
```diff
@@ -14,7 +14,7 @@ export const CaseMessagesIndividualInbox = connect(
           <thead>
             <tr>
               <th className="small" colSpan="2">
-                Docket Number
+                Docket No.
```
Comment (kkoskelin) [inline web-client/src/views/Messages/CaseMessagesIndividualInbox.jsx:17]: *aria-label

File: web-client/src/views/Messages/CaseMessagesIndividualOutbox.jsx [hunk 8]
```diff
@@ -14,7 +14,7 @@ export const CaseMessagesIndividualOutbox = connect(
           <thead>
             <tr>
               <th className="small" colSpan="2">
-                Docket Number
+                Docket No.
```
Comment (kkoskelin) [inline web-client/src/views/Messages/CaseMessagesIndividualOutbox.jsx:17]: *aria-label

File: web-client/src/views/Messages/CaseMessagesSectionCompleted.jsx [hunk 9]
```diff
@@ -13,7 +13,7 @@ export const CaseMessagesSectionCompleted = connect(
           <thead>
             <tr>
               <th className="small" colSpan="2">
-                Docket Number
+                Docket No.
```
Comment (kkoskelin) [inline web-client/src/views/Messages/CaseMessagesSectionCompleted.jsx:16]: *aria-label

File: web-client/src/views/Messages/CaseMessagesSectionInbox.jsx [hunk 10]
```diff
@@ -13,7 +13,7 @@ export const CaseMessagesSectionInbox = connect(
           <thead>
             <tr>
               <th className="small" colSpan="2">
-                Docket Number
+                Docket No.
```
Comment (kkoskelin) [inline web-client/src/views/Messages/CaseMessagesSectionInbox.jsx:16]: *aria-label

File: web-client/src/views/Messages/CaseMessagesSectionOutbox.jsx [hunk 11]
```diff
@@ -13,7 +13,7 @@ export const CaseMessagesSectionOutbox = connect(
           <thead>
             <tr>
               <th className="small" colSpan="2">
-                Docket Number
+                Docket No.
```
Comment (kkoskelin) [inline web-client/src/views/Messages/CaseMessagesSectionOutbox.jsx:16]: *aria-label

File: web-client/src/views/PendingReport/PendingReportList.jsx [hunk 12]
```diff
@@ -47,11 +47,11 @@ export const PendingReportList = connect(
         >
           <thead>
             <tr>
-              <th>Docket number</th>
-              <th>Date filed</th>
-              <th>Case title</th>
-              <th>Filings and proceedings</th>
-              <th>Case status</th>
+              <th>Docket No.</th>
```
Comment (kkoskelin) [inline web-client/src/views/PendingReport/PendingReportList.jsx]: *aria-label

File: web-client/src/views/Public/TodaysOpinions.jsx [hunk 13]
```diff
@@ -29,7 +29,7 @@ export const TodaysOpinions = connect(
                 <tr>
                   <th aria-hidden="true" />
                   <th aria-hidden="true" />
-                  <th>Docket Number</th>
+                  <th>Docket No.</th>
```
Comment (kkoskelin) [inline web-client/src/views/Public/TodaysOpinions.jsx]: *aria-label

File: web-client/src/views/StyleGuide/Tables.jsx [hunk 14]
```diff
@@ -63,7 +63,7 @@ export const Tables = () => (
     <table className="usa-table work-queue subsection">
       <thead>
         <tr>
-          <th>Docket number</th>
+          <th>Docket No.</th>
```
Comment (kkoskelin) [inline web-client/src/views/StyleGuide/Tables.jsx]: *aria-label

File: web-client/src/views/TrialSessionDetail/AllCases.jsx [hunk 15]
```diff
@@ -21,12 +21,12 @@ export const AllCases = connect(
         >
           <thead>
             <tr>
-              <th>Docket number</th>
+              <th>Docket No.</th>
```
Comment (kkoskelin) [inline web-client/src/views/TrialSessionDetail/AllCases.jsx]: *aria-label

File: web-client/src/views/TrialSessionDetail/EligibleCases.jsx [hunk 16]
```diff
@@ -33,14 +33,14 @@ export const EligibleCases = connect(
         >
           <thead>
             <tr>
-              <th>Docket number</th>
+              <th>Docket No.</th>
```
Comment (kkoskelin) [inline web-client/src/views/TrialSessionDetail/EligibleCases.jsx]: *aria-label

File: web-client/src/views/TrialSessionDetail/InactiveCases.jsx [hunk 17]
```diff
@@ -20,10 +20,10 @@ export const InactiveCases = connect(
         >
           <thead>
             <tr>
-              <th>Docket number</th>
-              <th>Case title</th>
+              <th>Docket No.</th>
```
Comment (kkoskelin) [inline web-client/src/views/TrialSessionDetail/InactiveCases.jsx]: *aria-label

File: web-client/src/views/TrialSessionDetail/OpenCases.jsx [hunk 18]
```diff
@@ -21,11 +21,11 @@ export const OpenCases = connect(
         >
           <thead>
             <tr>
-              <th>Docket number</th>
+              <th>Docket No.</th>
```
Comment (kkoskelin) [inline web-client/src/views/TrialSessionDetail/OpenCases.jsx]: *aria-label

File: web-client/src/views/TrialSessionWorkingCopy/WorkingCopySessionList.jsx [hunk 19]
```diff
@@ -65,15 +65,15 @@ export const WorkingCopySessionList = connect(
                       sort === 'docket' && 'sortActive',
                     )}
                   >
-                    Docket number
+                    Docket No.
```
Comment (kkoskelin) [inline web-client/src/views/TrialSessionWorkingCopy/WorkingCopySessionList.jsx:68]: *aria-label

File: web-client/src/views/WorkQueue/IndividualWorkQueueInProgress.jsx [hunk 20]
```diff
@@ -22,16 +22,16 @@ export const IndividualWorkQueueInProgress = connect(
           <thead>
             <tr>
               <th aria-label="Docket Number" colSpan="2">
-                <span className="padding-left-2px">Docket number</span>
+                <span className="padding-left-2px">Docket No.</span>
```
Comment (kkoskelin) [inline web-client/src/views/WorkQueue/IndividualWorkQueueInProgress.jsx:25]: *aria-label

File: web-client/src/views/WorkQueue/IndividualWorkQueueInbox.jsx [hunk 21]
```diff
@@ -20,16 +20,16 @@ export const IndividualWorkQueueInbox = connect(
           <thead>
             <tr>
               <th aria-label="Docket Number" className="small" colSpan="2">
-                <span className="padding-left-2px">Docket number</span>
+                <span className="padding-left-2px">Docket No.</span>
```
Comment (kkoskelin) [inline web-client/src/views/WorkQueue/IndividualWorkQueueInbox.jsx:23]: *aria-label

File: web-client/src/views/WorkQueue/IndividualWorkQueueOutbox.jsx [hunk 22]
```diff
@@ -24,24 +24,24 @@ export const IndividualWorkQueueOutbox = connect(
           <thead>
             <tr>
               <th aria-label="Docket Number" className="small" colSpan="2">
-                <span className="padding-left-2px">Docket number</span>
+                <span className="padding-left-2px">Docket No.</span>
```
Comment (kkoskelin) [inline web-client/src/views/WorkQueue/IndividualWorkQueueOutbox.jsx:27]: *aria-label

**Review by kkoskelin — CHANGES_REQUESTED** [review]
It may be the case that I marked a few items that already had the `aria-label` in them, in which case I apologize.  Lots of files changed!

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**